lib/client/rpc/openbookMarkets.ts

```diff
@@ -201,10 +177,10 @@ export class FutarchyOpenbookMarketsRPCClient
   ): Promise<string[]> {
     if (!openOrdersAccountAddress) {
       // use the first open orders account by default
-      const openOrdersIndexer = await this.getUsersOpenOrderPks(
+      const openOrdersIndexer = this.openbookClient.findOpenOrdersIndexer(
```

Comment:
I don't think this will work. This is not an OpenOrders account, it is an OpenOrdersIndexer account. They are not the same and I think this will break.

Reply:
looks like this is fixed. Thanks!